### PR TITLE
Tiny-Js-World

### PR DESCRIPTION
**Pull request by ArthurGorbenko**
### Tiny-Js-World

[Demo](https://arthurgorbenko.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/ArthurGorbenko/a-tiny-JS-world/blob/gh-pages/index.js)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by Roka20012** [comment]
Hi good work :+1:  
Left here only index.js please

**Comment by ArthurGorbenko** [comment]
@Roka20012 Thanks, can you review and suggest any improvements?

File: submissions/ArthurGorbenko/tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+print(objectsToString(dog));
+print(objectsToString(cat));
+print(objectsToString(man));
+print(objectsToString(woman));
+print(objectsToString(womanCat));
+
+// ======== OUTPUT ========
+/* Use print(message) for output.
```
Comment (aymkin) [inline submissions/ArthurGorbenko/tiny-js-world/index.js]: Please remove all comments and unrelated text.

File: submissions/ArthurGorbenko/tiny-js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        continue;
+        }
+        objectAsString += obj[prop] + ", ";
+    }
+    return objectAsString + ";";
+}
+
+print(objectsToString(dog));
+print(objectsToString(cat));
+print(objectsToString(man));
```
Comment (aymkin) [inline submissions/ArthurGorbenko/tiny-js-world/index.js]: `[dog, cat, man, woman, womanCat].forEach(inhobitant => print(objectToString(inhobitant)))`

this should optimize calling `print` function

File: submissions/ArthurGorbenko/tiny-js-world/index.js (hunk 1 above)
Comment (ArthurGorbenko) [reply submissions/ArthurGorbenko/tiny-js-world/index.js]: Done.

File: submissions/ArthurGorbenko/tiny-js-world/index.js (hunk 2 above)
Comment (ArthurGorbenko) [reply submissions/ArthurGorbenko/tiny-js-world/index.js]: Now it looks better.

**Comment by ArthurGorbenko** [comment]
@aymkin Thanks for you comments,please,check changes.

File: submissions/ArthurGorbenko/tiny-js-world/index.js (hunk 1 above)
Comment (aymkin) [reply submissions/ArthurGorbenko/tiny-js-world/index.js]: resolve it

File: submissions/ArthurGorbenko/tiny-js-world/index.js (hunk 2 above)
Comment (aymkin) [reply submissions/ArthurGorbenko/tiny-js-world/index.js]: resolve it

File: submissions/ArthurGorbenko/tiny-js-world/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        break;
+      } else if (prop === "hasTail" || prop === "legs") {
+        objectAsString += prop + " : " + obj[prop] + ", ";
+        continue;
+      }
+      objectAsString += obj[prop] + ", ";
+    }
+    return objectAsString + ";";
+  }
+
```
Comment (aymkin) [inline submissions/ArthurGorbenko/tiny-js-world/index.js]: Looks like there is no newline at the end of the file
line

Comment (ArthurGorbenko) [reply submissions/ArthurGorbenko/tiny-js-world/index.js]: Finally did ;)

**Comment by ArthurGorbenko** [comment]
Thanks for your time.

**Pull request merged**